CODEOWNERS

```diff
@@ -1001,6 +1001,7 @@ build.json @home-assistant/supervisor
 /tests/components/rmvtransport/ @cgtobi
 /homeassistant/components/roku/ @ctalkington
 /tests/components/roku/ @ctalkington
+/homeassistant/components/romy/ @xeniter
```

Comment:
I believe you need tests in this file as well, did you manually add this? It should be auto generated